src/web/content/dynamic/searchmysite/templates/admin/login.html

```diff
@@ -31,7 +31,7 @@
 							<label for="password" class="visually-hidden">Password</label>
 							<input type="password" name="password" id="password" class="form-control sms-b-16" placeholder="Password" required>
 							<button type="submit" class="btn btn-primary">Login with password</button>
-							<div class="text-center"><small><a class="nav-link" href="{{ url_for('auth.forgottenpassword_get') }}">Forgot password</a></small></div>
+							<div class="text-center"><small><a class="nav-link sms-forgot-pass-lnk" href="{{ url_for('auth.forgottenpassword_get') }}">Forgot password</a></small></div>
```

Comment:
Adding more specificity to fix an underline issue for this link. The underline had the full width of the container box, which was much larger than the words.